ledger/src/snapshot_utils.rs

```diff
@@ -354,8 +355,6 @@ pub fn add_snapshot<P: AsRef<Path>>(
     bank: &Bank,
     snapshot_storages: &[SnapshotStorage],
 ) -> Result<SlotSnapshotPaths> {
-    bank.clean_accounts();
```

Comment:
Cool, this is a nice fix, I know this was a [security issue](https://github.com/solana-labs/solana/issues/7167#issuecomment-590767412) for a while...